composer.json

```diff
@@ -20,7 +20,8 @@
 		"composer/installers": "~1.0",
 		"monolog/monolog": "~1.11",
 		"league/flysystem": "~1.0.12",
-		"symfony/yaml": "~2.7"
+		"symfony/yaml": "~2.7",
+		"guzzlehttp/guzzle": "~5.3|~6.0.1|~6.1"
```

Comment:
There are no breaking changes for this, I choose this, to comply with DynamoDB module, which is going to require AWS SDK 3, which has the same requirements.

As it was a hackday implementation, just copy-pasting the AWS-SDK versions was the fastest way to get it out. I guess just ^6.2 might even work.